physics/SFC_Layer/UFS/sfc_diff.f

```diff
@@ -163,6 +168,7 @@ subroutine sfc_diff_run (im,rvrdm1,eps,epsm1,grav,                &  !intent(in)
       errmsg = ''
       errflg = 0
 
+
```

Comment:
@binli2337, please revert this empty line change. Thanks!

Reply:
Done